src/permission.svc.js

```diff
@@ -121,6 +122,13 @@
 
             return definedPermissions;
           },
+          defineDefaultPermissions: function (permissions) {
+            defaultPermissions = permissions;
+            return Permission;
+          },
+          defaultPermissions: function () {
```

Comment:
Not sure if I would create a getter method for this. Why not just use the `defaultPermissions` var in the scope?